save_matrices.ipynb

```diff
@@ -0,0 +1,82 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from scipy import sparse\n",
+    "from neo4j.v1 import GraphDatabase\n",
+    "import hetio.readwrite\n",
+    "import hetio.neo4j\n",
+    "import hetio.pathtools\n",
+    "from hetmech.degree_weight import dwpc"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "url = 'https://github.com/dhimmel/hetionet/raw/76550e6c93fbe92124edc71725e8c7dd4ca8b1f5/hetnet/json/hetionet-v1.0.json.bz2'\n",
+    "graph = hetio.readwrite.read_graph(url)\n",
+    "metagraph = graph.metagraph"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for i in ['CCpGeAeGaD', 'DaGpBP', 'MFpGdCcSE']:\n",
```

Comment:
Should this match what's in the other notebook:

```
CCpGeAeGaD', 'DaGpBP', 'MFpGdCcSE', 'GiGpBP'
```

***

NVM, I see GiGpBP is done below. Can you add a comment why this metapath is processed separately?